### DISCUSSION TITLE
Does antd v5 css in js solution uses runtime styling?

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#45438] [Q&A]
In comparson to css in js solutions like Styled component or emotion which are runtime styling solution and have performance constraint and doesnt work in RSC. How does Antd's different from them?

:)

**Comment by zombieJ** [comment]
You can ref our blog for detail:
* https://ant.design/docs/blog/css-in-js
* https://ant.design/docs/blog/extract-ssr

**Comment by AsuraKev** [comment]
Awesome! is Antd css in js also working in RSC ?
